Github Release task for Windows can't find cmake

During stage `Configure Cmake` There is error:

```
Run cmake -DCMAKE_BUILD_TYPE=Release -DCHECK_FOR_UPDATES=ON -DANTIMICROX_PKG_VERSION="GitHub Windows Release" -B 'D:\a\antimicrox\antimicrox'/build
D:\a\_temp\5f82122d-569c-41ef-82a3-c9baedf3fdf5: line 1: cmake: command not found
Error: Process completed with exit code 127.
```

There is also another problem

```
CPack: Create package
CPack Error: Problem running NSIS command: "C:/msys64/mingw64/bin/makensis.exe" "D:/a/antimicrox/antimicrox/build/_CPack_Packages/win64/NSIS/project.nsi"
Please check D:/a/antimicrox/antimicrox/build/_CPack_Packages/win64/NSIS/NSISOutput.log for errors
CPack Error: Problem compressing the directory
CPack Error: Error when generating package: antimicrox
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -749,7 +749,7 @@ if(WIN32 AND NOT UNIX)
 
     # There is a bug in NSI that does not handle full UNIX paths properly.
     # Make sure there is at least one set of four backlashes.
-    set(CPACK_PACKAGE_ICON "${PROJECT_SOURCE_DIR}/src/icons\\\\antimicrox.ico")
+    set(CPACK_PACKAGE_ICON "${PROJECT_SOURCE_DIR}/src/images\\\\antimicrox.ico")
     set(CPACK_NSIS_MUI_ICON "${CPACK_PACKAGE_ICON}")
     set(CPACK_NSIS_MUI_UNIICON "${CPACK_PACKAGE_ICON}")
     set(CPACK_NSIS_INSTALLED_ICON_NAME "bin\\antimicrox.exe")

